schema/sale_artwork.js

```diff
@@ -215,16 +218,27 @@ const SaleArtworkType = new GraphQLObjectType({
         type: GraphQLInt,
         deprecationReason: 'Favor `counts.bidder_positions`',
       },
-      bid_increment: {
-        type: GraphQLInt,
+      bid_increments: {
+        type: new GraphQLList(GraphQLInt),
         resolve: ({ minimum_next_bid_cents }) => {
           return gravity('/increments').then((res) => {
             const deflt = find(res, { key: 'default' });
-            const bucket = find(deflt.increments, (inc) =>
-              minimum_next_bid_cents >= inc.from &&
-              minimum_next_bid_cents <= inc.to
-            );
-            return bucket.amount;
+            const increments = [minimum_next_bid_cents];
```

Comment:
Ooo, a puzzler. How about:

```
function increments(from, to) {
  function loop(amount, [tier, ...restTiers]) {
    let nextAmount, nextTiers, nextRestTiers;
    if (amount + tier.amount > tier.to) {
      if (restTiers.length == 0) return [];
      [nextTier, ...nextRestTiers] = restTiers;
      nextAmount = amount + newTier.amount;
    } else {
      nextTier = tier;
      restTiers = restTiers;
      nextAmount = amount + tier.amount;
    }
    if (nextAmount > to) return [];
    return [nextAmount, ...loop(nextAmount, nextTier, ...nextRestTiers)];
  }
  return loop(from, deflt.increments);
}
```

No idea if that works!

I think the original idea was to make the top of the range some multiple of the max of the current price and the high estimate. But an arbitrary top end should suffice for now!